src/rgw/rgw_perf_counters.h

```diff
@@ -58,3 +52,52 @@ enum {
   l_rgw_last,
 };
 
+enum {
+  l_rgw_op_first = 16000,
+
+  l_rgw_op_put,
+  l_rgw_op_put_b,
+  l_rgw_op_put_lat,
+
+  l_rgw_op_get,
+  l_rgw_op_get_b,
+  l_rgw_op_get_lat,
+
+  l_rgw_op_del_obj,
+  l_rgw_op_del_obj_b,
+  l_rgw_op_del_obj_lat,
+
+  l_rgw_op_del_bucket,
+  l_rgw_op_del_bucket_lat,
+
+  l_rgw_op_copy_obj,
+  l_rgw_op_copy_obj_b,
+  l_rgw_op_copy_obj_lat,
+
+  l_rgw_op_list_obj,
+  l_rgw_op_list_obj_lat,
+
+  l_rgw_op_list_buckets,
+  l_rgw_op_list_buckets_lat,
+
+  l_rgw_op_last
+};
+
+namespace rgw::op_counters {
+
+extern PerfCounters *global_op_counters;
```

Comment:
please remove this global variable from the header to force callers to use `get()` and `inc()` instead